GEOSldas_App/setup_utils.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+from collections import OrderedDict
+from datetime    import timedelta
+
+
+def generate_echo(inpfile, ladas_cpl = 0):
+   """
+    Echo generator of inpfile, ignore line starts with "## "
+   """
+   if ladas_cpl == 0 :
+      use_rc_defaults = '# GEOSldas=>'    # use defaults for LDAS
+   else :
+      use_rc_defaults = '# GEOSagcm=>'    # use defaults for AGCM
+
+   with open (inpfile) as fin :
+     for line in fin:
+        if line.startswith(use_rc_defaults):
+           line = line.split(use_rc_defaults)[1]
```

Comment:
Why can't we replace this with the following and avoid the "startswith"?

```suggestion
       if use_rc_defaults in line:
          line = line.split(use_rc_defaults)[1]
```

Reply:
Combined with `use_rc_defaults = 'GEOSldas=>'`,  this would let us avoid having the '# GEOS[xxxx]=>' format with the all-important single whitespace character being in just the right place.

Reply:
if so, we need special codes to deal with  this line https://github.com/GEOS-ESM/GEOSgcm_GridComp/blob/a6a6f9b66c452e1afb5d1c32c12e5767a2c468b8/GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/Shared/GEOS_SurfaceGridComp.rc#L10

Reply:
Are we waiting for this PR https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1134 ?

Reply:
@weiyuan-jiang : Yes, except the relevant PR is https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1133. Scott said he will merge https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1133 (and https://github.com/GEOS-ESM/GEOSgcm_GridComp/pull/1134) as soon as the CI test completes.  But there seems to be an issue with CI that cannot be related to the PRs.  I assume it will get sorted soon.  Let's wait for now.